docs/GettingStartedDocs/Contributors/SGX1FLCGettingStarted.md

```diff
@@ -40,6 +40,8 @@ If you are not running in an ACC VM, you should instead run:
 ansible-playbook scripts/ansible/oe-contributors-setup.yml
 ```
 
+To learn more about setting up Open Enclave SGX on Linux in a Non-Azure Confidential Computing machine, read the document [Configure OE SDK SGX on Linux in non-ACC Machines](./NonAccMachineSGXLinuxGettingStarted.md).
```

Comment:
Don't you also need to add this to SGX1GettingStarted.md ?

Reply:
Thanks for asking. SGX DCAP attestation requires Flexible Launch Control (FLC) feature which is available since 8th gen core (CoffeeLake). So we only need to add this doc's link to SGX1FLCGettingStarted.md, no need to add it to SGX1GettingStarted.md.